### PR TITLE
Update project root files based on updated templates

### PR DESCRIPTION
**Pull request by rjeberhard**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,28 +1,55 @@
...
+
+1. Ensure there is an issue created to track and discuss the fix or enhancement
+   you intend to submit.
 1. Fork this repository
-2. Create a branch in your fork to implement the changes. We recommend using the issue number as part of your branch name, e.g. 1234-fixes
-3. Ensure that any documentation is updated with the changes that are required by your fix.
-4. Ensure that any samples are updated if the base image has been changed.
-5. Squash your branch changes into a single commit.
-6. Submit the pull request. Do not leave the pull request blank. Explain exactly what your changes are meant to do and provide simple steps on how to validate your changes. Ensure that you reference the issue you created as well. We will assign someone to review the pull request before it is merged.
+1. Create a branch in your fork to implement the changes. We recommend using
```
Comment (rakillen) [inline CONTRIBUTING.md:39]: the numbering messed up here

File: LICENSE.txt [hunk 2]
```diff
@@ -33,58 +33,3 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
-
```
Comment (rakillen) [inline LICENSE.txt:36]: are these licenses still needed, or did they move somewhere else?
the project still uses Antlr, not sure why codepoint is included.

File: CONTRIBUTING.md (hunk 1 above)
Comment (rjeberhard) [reply CONTRIBUTING.md:39]: The numbers don't actually matter for a markdown file. You can just use "1" over and over as here.

Comment (rjeberhard) [reply CONTRIBUTING.md:39]: You can validate by looking at the generated file here: https://github.com/oracle/weblogic-deploy-tooling/blob/update-template/CONTRIBUTING.md

File: LICENSE.txt (hunk 2 above)
Comment (rakillen) [reply LICENSE.txt:36]: Looks like these were moved to THIRD_PARTY_LICENSES.txt, so all good.

**Review by rakillen — APPROVED** [review]

**Review by ddsharpe — APPROVED** [review]

**Pull request merged**